docs/examples/te_llama/tutorial_accelerate_hf_llama_with_te.ipynb

```diff
@@ -0,0 +1,968 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ffdcbbb5",
+   "metadata": {},
+   "source": [
+    "# Accelerate HF Llama model with TransformerEngine\n",
+    "\n",
+    "<div class=\"alert alert-success\">\n",
+    "\n",
+    "<b>Goal</b>\n",
+    "\n",
+    "This tutorial showcases three incrementally efficient ways to use [TransformerEngine library](https://github.com/NVIDIA/TransformerEngine) to finetune (full) a Llama2 model from [Hugging Face](https://huggingface.co/meta-llama/Llama-2-7b-hf).\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0a85813",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "<b>Note</b>\n",
+    "    \n",
+    "This tutorial showcases finetuning a full 7B [Llama2 model](https://huggingface.co/meta-llama/Llama-2-7b-hf) on h100 GPUs (which have 80GB of HBM). Therefore, running the following individual portions (Baseline, Improvement 1, Improvement 2 and Improvement 3) for perf benchmarking will require restarting the Jupyter notebook kernel each time.\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d20a6fc8",
+   "metadata": {},
+   "source": [
+    "## Table of contents\n",
+    "1. From \"Transformer\" to \"Llama\"\n",
+    "2. Hugging Face's `LlamaModel`\n",
+    "    - Hugging Face's `LlamaDecoderLayer`\n",
+    "3. [Baseline] Running HF `LlamaModel` (Precision: `BF16`)\n",
+    "4. [Improvement 1] Replace `nn.Linear` with TE's `Linear` layers (Precision: FP8)\n",
+    "    - Understanding the aberration with Improvement 2 (CPU Overheads)\n",
+    "5. [Improvement 2] Replace HF's `LlamaDecoderLayer` with TE's `TransformerLayer` (Precision: `BF16`)\n",
+    "    - TransformerEngine's `TransformerLayer`\n",
+    "    - `TransformerLayer` options explained\n",
+    "    - Comparing Hugging Face's `LlamaDecoderLayer` with TranformerEngine's `TransformerLayer`\n",
+    "    - Mapping weights from HF's `LlamaDecoderLayer` to TE's `TransformerLayer`\n",
+    "6. [Improvement 3] Replace HF's `LlamaDecoderLayer` with TE's `TransformerLayer` (Precision: `FP8`)\n",
+    "7. Benchmarks revisited and Conclusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "82113cd0",
+   "metadata": {},
+   "source": [
+    "## From \"Transformer\" to \"Llama\" \n",
+    "\n",
+    "<figure align=\"center\">\n",
+    "<img src=\"media/transformer_llama.png\" width=\"50%\">\n",
+    "    <figcaption> Fig 1: Llama visualized as a transformer. (generated with <a href=\"https://catalog.ngc.nvidia.com/orgs/nvidia/teams/ai-foundation/models/sdxl\">Nvidia's AI-foundation models</a>)</figcaption>\n",
+    "</figure>\n",
+    "\n",
+    "A flashback:\n",
```

Comment:
This is not a valid RST list - you need to have an empty line before the start of the list and after the end. Otherwise it will not render properly as HTML: 
![image](https://github.com/NVIDIA/TransformerEngine/assets/8398980/920d102d-4264-4cd2-ad71-b054ba7d74e8)